### PR TITLE
fix(Tabs): Add withDivider property

### PR DESCRIPTION
**Pull request by wichniowski**
This PR adds a withDivider property to the Tabs component. if this is true a border will be shown underneath the tabs panel. 

image

### CODE REVIEW

File: packages/forma-36-react-components/src/components/Tabs/Tabs.css [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 .Tabs {
   display: flex;
+  padding: 0 var(--spacing-l);
```
Comment (wichniowski) [inline packages/forma-36-react-components/src/components/Tabs/Tabs.css]: @m10l @joaoramos  I added some extra padding left and right to visually balance the tab panel but I'm not sure if it is needed. Any thoughts on that?

File: packages/forma-36-react-components/src/components/Tabs/Tabs.css [hunk 2]
```diff
@@ -30,6 +30,10 @@
   margin-right: 0;
 }
 
+.Tabs__with-divider {
+  border-bottom: 1px solid var(--color-element-mid);
```
Comment (m10l) [inline packages/forma-36-react-components/src/components/Tabs/Tabs.css]: The following looks quite nice as an alternative, but it's a bigger change and maybe doesn't match anything else in the UI :D

```
box-shadow: 0 -3px 0 inset var(--color-element-mid);
```

Screenshot 2019-08-21 at 16 07 40

File: packages/forma-36-react-components/src/components/Tabs/Tabs.stories.tsx [hunk 3]
```diff
@@ -12,7 +12,10 @@ function DefaultStory() {
 
   return (
     <div>
-      <Tabs className={text('className', '')}>
+      <Tabs
+        className={text('className', '')}
+        withDivider={boolean('withDivider', false)}
```
Comment (m10l) [inline packages/forma-36-react-components/src/components/Tabs/Tabs.stories.tsx]: Nitpick but would be nice to add this to the second Tabs story too

File: packages/forma-36-react-components/src/components/Tabs/Tabs.css (hunk 2 above)
Comment (wichniowski) [reply packages/forma-36-react-components/src/components/Tabs/Tabs.css]: ahh that is what you meant that looks nice will do so!

Comment (wichniowski) [reply packages/forma-36-react-components/src/components/Tabs/Tabs.css]: @m10l done

**Pull request merged**